srcpkgs/recutils/patches/copyfile.patch

```diff
@@ -0,0 +1,37 @@
+--- a/utils/recutl.c
```

Comment:
Add a comment on top explaining the purpose of this patch

Reply:
Done, sorry :D